pyi.py

```diff
@@ -643,6 +643,17 @@ def visit_ClassDef(self, node: ast.ClassDef) -> None:
                 self.error(statement, Y013)
 
     def visit_FunctionDef(self, node: ast.FunctionDef) -> None:
+        if (
+            self.in_class
+            and node.name in {"__repr__", "__str__"}
+            and isinstance(node.returns, ast.Name)
+            and node.returns.id == "str"
```

Comment:
Should also check that there are no extra arguments, in case somebody decides that it's a good idea to override `__str__` and add more arguments.

Reply:
Done!